### DISCUSSION TITLE
导入插件时搜索不到

### DISCUSSION THREAD

**Discussion by Std1oh** [nonebot/discussions#235] [问答]
萌新求助，nb plugin install时搜索不到对应插件。
![屏幕截图 2022-11-11 002005](https://user-images.githubusercontent.com/101545666/201150065-a368716d-fe6d-4d69-9cbb-75ed273ec5a4.png)

**Comment by yanyongyu** [comment]
网络问题，黄字都写了

**Reply by Std1oh** [reply]
感谢，因为第一次用的腾讯云服务器，我去调一下网络配置。

>
